content/docs/installation/helm.md

```diff
@@ -7,9 +7,6 @@ description: 'cert-manager installation: Using Helm'
 
 cert-manager provides Helm charts as a first-class method of installation on both Kubernetes and OpenShift.
 
-Be sure never to embed cert-manager as a sub-chart of other Helm charts; cert-manager manages
-non-namespaced resources in your cluster and care must be taken to ensure that it is installed exactly once.
```

Comment:
I suggest that we adjust this message and move it above the new sub-charts section e.g.

https://cert-manager.io/docs/installation/helm/#installing-cert-manager-as-subchart

## Installing cert-manager as subchart

> :warning: If you embed cert-manager as a sub-chart care must be taken to ensure that it is installed exactly once. cert-manager manages non-namespaced resources such as ClusterIssuer which will break if there are multiple cert-manager controllers installed. Likewise, the cert-manager webhook and cainjector components must only be installed once.